.drone.yml

```diff
@@ -82,7 +82,7 @@ steps:
   - name: unit-tests
     # see: docker/ci/Dockerfile
     # TODO: switch to codedotorg/cdo-ci
-    image: codedotorg/code-dot-org:1.14
+    image: codedotorg/code-dot-org:chrome-for-testing
```

Comment:
upon approval, I'll retag this image as `code-dot-org:1.15` and update and sign `.drone.yml`